packages/amplify-e2e-core/src/index.ts

```diff
@@ -111,14 +112,16 @@ export async function createNewProjectDir(
   // eslint-disable-next-line spellcheck/spell-checker
   prefix = path.join(fs.realpathSync(os.tmpdir()), amplifyTestsDir),
 ): Promise<string> {
-  const currentHash = execSync('git rev-parse --short HEAD', { cwd: __dirname }).toString().trim();
   let projectDir;
   do {
-    const randomId = await global.getRandomId();
-    projectDir = path.join(prefix, `${projectName}_${currentHash}_${randomId}`);
+    projectDir = path.join(prefix, `${projectName}_${Math.floor(Math.random() * 1000000)}`);
   } while (fs.existsSync(projectDir));
 
   fs.ensureDirSync(projectDir);
+
+  const initialDelay = Math.floor(Math.random() * 180 * 1000);
+  await sleep(initialDelay);
```

Comment:
createProjectDir(..) is something that nearly every test uses; and we need a way to stagger the tests to avoid hotspots on a specific account+region.
This makes it a perfect candidate for staggering test start times, which I had previously attempted using addCircleCITags(..), but I found that method was over complicated- and this does the job just fine.